package.json

```diff
@@ -77,6 +88,8 @@
     "xmlbuilder2": "^2.4.1"
   },
   "dependencies": {
+    "@aws-sdk/credential-provider-node": "^3.154.0",
```

Comment:
So does this become a hard dependency for this package? Meaning if I am not doing AWS sigv4, am I still dragging this module in? Asking again because I'd like not to drag any vendor specific components in by default.

Reply:
Removed this dependency like I mentioned below. We are still depending on the aws4 library which we cannot do away(unless we want to re-write all of that implementation) if we want to support signing natively in the client.